pkg/controller/crdmirroring/crdmirroring_tracker.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2021 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package crdmirroring
+
+import (
+	"sync"
+)
+
+type tracker struct {
+	lock     sync.Mutex
+	mirrored map[string]bool
+}
+
+func newTracker() *tracker {
+	return &tracker{
+		mirrored: map[string]bool{},
+	}
+}
+
+func (t *tracker) add(name string) {
```

Comment:
Not all resources are cluster scoped, for example the AntreaNetworkPolicy. Does it take it into consideration?